app/src/main/java/com/example/kdotz/contactmgr/ContactDetailActivityFragment.java

```diff
@@ -23,10 +23,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
         Intent intent = getActivity().getIntent();
 
-        String item = intent.getStringExtra("item");
+        String name = intent.getStringExtra(ContactDetailActivity.EXTRA_NAME);
+        String number = intent.getExtras().getString(ContactDetailActivity.EXTRA_NUMBER);
```

Comment:
Normally you'd want to do null checks here to make sure these items exist (they could be null). You can also get default values in the getString call